src/cow-react/modules/limitOrders/pure/ReceiptModal/styled.ts

```diff
@@ -35,138 +34,152 @@ export const Header = styled.div`
   flex-direction: row;
   justify-content: space-between;
   align-items: center;
-  padding: 1rem;
-  padding-top: 0;
+  padding: 0 16px 16px;
 `
 
 export const Title = styled.h3`
-  font-size: 1rem;
+  font-size: 16px;
   margin: 0;
 `
 
 export const Body = styled(AutoRow)`
   box-sizing: border-box;
-  max-height: 700px;
-  padding: 1rem;
-  padding-top: 0;
-
-  ${({ theme }) => theme.mediaWidth.upToMedium`
-    max-height: 70vh;
-  `}
+  max-height: 70vh;
+  padding: 0 10px 10px;
 
   ${({ theme }) => theme.mediaWidth.upToSmall`
     max-height: 100%;
   `}
 `
 
-export const Field = styled.div<{ border?: 'rounded-full' | 'rounded-top' | 'rounded-bottom' }>`
+export const FieldsWrapper = styled.div`
+  display: flex;
+  flex-flow: column wrap;
+  width: 100%;
+  gap: 2px;
+`
+
+export const Field = styled.div`
   display: flex;
   flex-flow: row wrap;
   justify-content: space-between;
   align-items: center;
-  padding: 1rem;
-  background: ${({ theme }) => theme.input.bg1};
-  margin-bottom: 5px;
+  padding: 12px 16px;
```

Comment:
I would do here just 16px, but again just a personal preference.

Reply:
By doing 12px for top/bottom, we 'gain 8px per Field (4px top and 4px bottom difference), which for 10 Fields = 80 precious pixelito's in total 😅  - Point being that may prevent some early scrolling. Appreciate the review.